React/Extensions/HtmlHelperExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.DependencyInjection;
+using React.Services;
+
+namespace React.Extensions;
```

Comment:
please consider moving the extension method to the namespace of the extended type as this is good practice from a usage perspective